api/printful/printful.php

```diff
@@ -26,23 +26,23 @@ protected function send_request($URI, $data = false) {
 
         curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
 
-        $result = curl_exec($ch);
+        $cache_result = get_valid_cache_entry($endpoint, $ch, $this::PRINTFUL_CACHE_CONFIG_ID);

```

Comment:
Aside from the config ID being provided here, this is exactly what I was looking for 👍